datasets/glossis/vtkDockerfile

```diff
@@ -0,0 +1,21 @@
+FROM continuumio/miniconda3
```

Comment:
Like you said, we could merge this together with the other Dockerfile. Might be a good improvement. We could also add an AUTHOR and some metadata in this Dockerfile.

Reply:
yes, but maybe improve this in a new pull request